src/log.js

```diff
@@ -219,6 +219,7 @@ export class Log {
    * @param {...*} var_args Arguments substituted into %s in the message.
    * @return {T} The value of shouldBeTrueish.
    * @template T
+   * @throws {Error}
```

Comment:
Are you sure it is being used at all? I don't think so. Probably on externs
only.

On Jul 28, 2016 10:21 AM, "erwin mombay" <notifications@github.com> wrote:

> In src/log.js
> <https://github.com/ampproject/amphtml/pull/4249#discussion_r72581746>:
>
> > @@ -219,6 +219,7 @@ export class Log {
> >     * @param {...*} var_args Arguments substituted into %s in the message.
> >     * @return {T} The value of shouldBeTrueish.
> >     * @template T
> > +   * @throws {Error}
>
> should i remove here then? didnt realize the compiler doesnt even use it
> besides detecting side effects
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4249/files/7ef905fb26fde6d8e3f030df76a6c780dba8d889#r72581746>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFeT6itAXu5wks1ywbO7YO617Kpzhzvks5qaGZ9gaJpZM4JWvb9>
> .
>